helix-term/src/commands.rs

```diff
@@ -310,8 +311,7 @@ impl MappableCommand {
         goto_line_end, "Goto line end",
         goto_next_buffer, "Goto next buffer",
         goto_previous_buffer, "Goto previous buffer",
-        // TODO: different description ?
-        goto_line_end_newline, "Goto line end",
+        goto_line_end_newline, "Goto line end (newline)",
         goto_first_nonwhitespace, "Goto first non-blank in line",
```

Comment:
Looks like an unrelated change?

Reply:
Might as well rename this though. I would prefer "Goto newline at line end"

Reply:
Apologies, I forgot I did that when reordering. I have updated to description.